### PR TITLE
Add statemine config

### PR DESCRIPTION
**Pull request by riusricardo**
Adds a new Statemine config to generate the chain spec.
`./target/release/statemint build-spec --chain statemine --disable-default-bootnode > statemine-plain.json`

### CODE REVIEW

File: node/src/chain_spec.rs [hunk 1]
```diff
@@ -312,7 +313,53 @@ pub fn statemine_local_config(id: ParaId) -> StatemineChainSpec {
...
+		None,
+		Some(properties),
+		Extensions {
+			relay_chain: "kusama".into(),
+			para_id: id.into(),
+		},
+	)
+}
+
+fn statemine_genesis(
```
Comment (apopiak) [inline node/src/chain_spec.rs:362]: we don't need any aura authorities?

File: node/src/cli.rs [hunk 2]
```diff
@@ -49,7 +49,7 @@ pub struct ExportGenesisStateCommand {
 
 	/// Id of the parachain this state is for.
 	///
-	/// Default: 1001
+	/// Default: 1000
 	#[structopt(long)]
```
Comment (apopiak) [inline node/src/cli.rs:53]: would probably be good to move the default into the structop declaration

Comment (apopiak) [inline node/src/cli.rs:53]:
```suggestion
	/// Default: 1000
	#[structopt(default_value = "1000", long)]
```

File: node/src/chain_spec.rs (hunk 1 above)
Comment (riusricardo) [reply node/src/chain_spec.rs:362]: Not here. It is handled by the collators selection pallet.

File: node/src/cli.rs (hunk 2 above)
Comment (riusricardo) [reply node/src/cli.rs:53]: We could add this change or just handle it as part of the command.
https://github.com/paritytech/statemint/pull/82/files#diff-8ce03a5a2b79f3021ff2d6322d13448db8cfd5fb9267517ab4b96d0d299ed45fR99

Comment (apopiak) [reply node/src/cli.rs:53]: Sure we can keep the `unwrap_or`s, but maybe extract into a constant?

**Pull request merged**